<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FtpDialog</class>
 <widget class="QDialog" name="FtpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>当前连接数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>125</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>当前目录：</string>
   </property>
  </widget>
  <widget class="QLabel" name="dir_lb">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>//</string>
   </property>
  </widget>
  <widget class="QLabel" name="count_lb">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QListWidget" name="file_lw">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>341</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="msg_lb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>消息</string>
   </property>
  </widget>
  <widget class="QLabel" name="count_lb_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>个</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ip</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IP：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cddir">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>改前当前目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_usr">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>查看客户</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
